packages/cashier/src/Containers/deposit.jsx

```diff
@@ -102,6 +104,9 @@ const Deposit = ({
     if (error.message) {
         return <Error error={error} />;
     }
+    if (is_crypto_transactions_visible) {
+        return <CryptoTransactionsHistory />;
```

Comment:
A question, As I see the design when the `transaction history` is open, the active tab is withdraw, so should we put this here in deposit or in withdraw? (and you add the component in `routes-config.js` under deposit also).